src/amo/css/App.scss

```diff
@@ -66,7 +66,7 @@ a:link {
   @include font-medium();
 
   font-size: 14px;
-  color: #0996f8;
+  color: #0568ba;
```

Comment:
I'm going to file a separate issue for this. Seems we don't have any way to set vars for just this app - not sure if that got removed or we never had it - I don't think we should only have core vars. That needs to be fixed and I don't want to block on that for this patch.